### DISCUSSION TITLE
Gogoanime Website renamed and glitching after update

### DISCUSSION THREAD

**Discussion by Dark-Knight987** [aniyomiorg/aniyomi#1217] [General]
This morning I was having issues accessing gogoanime untill I discovered that it was relocated at anitaku.to .I was a simple fix , go to the extension settings and override the link from https://gogoanime.hd to https://anitaku.to .since the structure of the website hasn't changed so it works .

[There has been a update that fixed the issue but now gogoanime is frequently timing out and mostly not even downloading with error code 502 which I believe is a bad server response or maybe issue with cors or captcha ,kindly fix this please.]

**Comment by rakdso** [comment]
I have changed to the new URL link, but none of the episodes of any anime is working it gives an error "couldn't find any video links"

**Reply by Dark-Knight987** [reply]
Well it works for me ,and I guess there is also an extension update now that has fixed the issue .

**Reply by rakdso** [reply]
made it work... I installed the new version of Aniyomi 12.3.10 and restarted the app it started working now...

**Reply by rakdso** [reply]
yes updated all... it is working now

**Reply by Prathmeshkasar007** [reply]
https://gogoanime.by/
is working

**Reply by Lushdoll69** [reply]
> https://gogoanime.by/ is working

Your definition on working is a pop up blocked add invested shit show

**Reply by g18089834-rgb** [reply]
https://anitaku.com.co/
is working ad-free site now

**Comment by Dark-Knight987** [comment]
![Screenshot_20231127-124922.jpg](https://github.com/aniyomiorg/aniyomi/assets/108930015/df342692-5f5b-4b1c-9b35-af9b5d11413d)

Here's the override url I've used.

**Reply by Nothing013** [reply]
I don't have the option to Override Baseurl
Please help me 🙏😭

**Reply by ShavonHastings** [reply]
How do u override the url?

**Comment by bestbreadmakerst** [comment]
Can I ask what Gogoanime's official homepage is? There are many websites with the same content as above.
![Screenshot_1](https://github.com/aniyomiorg/aniyomi/assets/170799964/494f22ac-95df-4b6c-9d5b-a0b049568978)

**Reply by Lushdoll69** [reply]
I use gogoanime.no  but it stopped letting me look at different pages like its one of the only ones that don't have a pop up add convention and let's u actually watch stuff like it was the only place that updated Chinese and Japanese anime with sub usually so now I'm looking for something as good as this was before shit stopped working

**Comment by Red-dev275** [comment]
Can I join

**Comment by Red-dev275** [comment]
I like anime

**Comment by Red-dev275** [comment]
![Screenshot_2024-06-26-16-45-17-546](https://github.com/aniyomiorg/aniyomi/assets/173081297/7669a2c5-9890-4e33-8c91-c44b086c9feb)

**Comment by Nothing013** [comment]
But I don't have the option to Override Baseurl
Please help me 🙏😭

**Comment by appslaib** [comment]
[https://anime-sama.com.ro/](https://anime-sama.com.ro/)
Yes is working ad-free site now
